release-notes.md

```diff
@@ -0,0 +1,60 @@
+# Release v0.9.0
+
+## New Features
+* Add pai service dashboard to grafana. - [PR 1694](https://github.com/Microsoft/pai/pull/1694)
+* Webportal plugin implementation and change pai marketplace to plugin. -[PR 1700](https://github.com/Microsoft/pai/pull/1700)
```

Comment:
Plugins doc: https://github.com/Microsoft/pai/blob/master/docs/webportal/PLUGINS.md

Reply:
Thanks @Gerhut , this is great help. Suggest to change the ReleaseNote as below:

Support to add custom web pages to the web portal of PAI deployments with **WebPortal Plugin**, refer to [Plugins Doc]( https://github.com/Microsoft/pai/blob/master/docs/webportal/PLUGINS.md) for how to use the new feature, and refer to [PR 1700](https://github.com/Microsoft/pai/pull/1700) for how PAI Marketplace is using it as an example.